src/components/mc/cpu/mc_cpu.h

```diff
@@ -12,11 +12,19 @@
 
 typedef struct ucc_mc_cpu_config {
     ucc_mc_config_t super;
+    size_t          cpu_elem_size;
```

Comment:
cpu_elem_size -> mpool_elem_size
cpu_max_elems->mpool_max_elems

(it is already located in mc/cpu and in MC_CPU_CONFIG so extra "cpu" is not necessary imho)